### PR TITLE
QOL update for manpages

### PR DESCRIPTION
**Pull request by TheRealKeto**
This PR provided QOL changes to all manpages. Specific changes have been documented under the commit message.

### CODE REVIEW

File: uicache.1 [hunk 1]
```diff
@@ -28,50 +28,47 @@
...
 .Op Fl p Ar path
 .Op Fl u Ar path
 .Sh DESCRIPTION
-The
-.Nm
-updates the applications that show on the homescreen and optionally will respring afterwards.
-.Pp
-The options are as follows:
+Update iOS registered applications and optionally restart SpringBoard.
+.Sh OPTIONS
```
Comment (CRKatri) [inline uicache.1:40]:
```suggestion
The following options are available:
```

File: uiopen.1 [hunk 2]
```diff
@@ -37,11 +37,9 @@
 .Op Fl p Ar path
 .Op Fl u Ar url
 .Sh DESCRIPTION
-The
 .Nm
-will open the specified app or url on SpringBoard.
-.Pp
-The options are as follows:
+will open a specified app or URL, displaying it on SpringBoard.
+.Sh OPTIONS
```
Comment (CRKatri) [inline uiopen.1:42]:
```suggestion
The following options are available:
```

File: uiopen.1 [hunk 3]
```diff
@@ -55,13 +53,13 @@ Opens the application at the specified .app bundle.
...
-To open github.com in the default browser use:
+To open github.com in the default browser
 .Dl "uiopen -u https://github.com"
 .Pp
-To open the Settings app by name use:
+To open the Settings app
 .Dl "uiopen -a Settings"
 .Pp
-To open the application at /Applications/Sileo.app use:
+To open Sileo using it's path, /Applications/Sileo.app
```
Comment (CRKatri) [inline uiopen.1:62]:
```suggestion
To open Sileo using it's path, /Applications/Sileo.app
```

```
% mandoc -Tlint -Wall *.1
mandoc: uiopen.1:62:55: STYLE: whitespace at end of input line
```

**Review by CRKatri — CHANGES_REQUESTED** [review]

File: uicache.1 (hunk 1 above)
Comment (CRKatri) [reply uicache.1:40]: This is because `.Sh OPTIONS` is not a standard section of manpages, so just to keep conformity I don't think it should be used.

Comment (TheRealKeto) [reply uicache.1:40]: This works for me. Maybe consider using another system; must GNU projects have this on their manpages, so it fits fine.

Comment (CRKatri) [reply uicache.1:40]: Ok.

**Review by CRKatri — APPROVED** [review]

**Comment by CRKatri** [comment]
@Diatrus when you merge this, re-tag 2.0.5, or tag 2.0.5-1.

File: cfversion.1 [hunk 4]
```diff
@@ -28,13 +28,13 @@
...
 .Nm cfversion
-.Nd print the installed CoreFoundation version to stdout
+.Nd print the CoreFoundation version of your device
 .Sh SYNOPSIS
 .Nm
 .Sh DESCRIPTION
 The
 .Nm
-utility will print the version of the installed CoreFoundation then exit.
+utility will print the version of the CoreFoundation of your device, then exit.
```
Comment (CRKatri) [inline cfversion.1:37]:
```suggestion
utility will print the version of the CoreFoundation on the device, then exit.
```

File: cfversion.1 [hunk 5]
```diff
@@ -28,13 +28,13 @@
 .Os
 .Sh NAME
 .Nm cfversion
-.Nd print the installed CoreFoundation version to stdout
+.Nd print the CoreFoundation version of your device
```
Comment (CRKatri) [inline cfversion.1:31]:
```suggestion
.Nd print the CoreFoundation version of the device
```

File: ecidecid.1 [hunk 6]
```diff
@@ -28,13 +28,12 @@
 .Os
 .Sh NAME
 .Nm ecidecid
-.Nd print your device ecid to stdout
+.Nd print your device's ECID
```
Comment (CRKatri) [inline ecidecid.1:31]:
```suggestion
.Nd print the device's ECID
```

File: ecidecid.1 [hunk 7]
```diff
@@ -28,13 +28,12 @@
...
 .Sh SYNOPSIS
 .Nm
 .Sh DESCRIPTION
-The
 .Nm
-utility will print the ecid then exit.
+will print the ECID of your device, then exit.
```
Comment (CRKatri) [inline ecidecid.1:36]:
```suggestion
will print the ECID of the device, then exit.
```

File: uiduid.1 [hunk 8]
```diff
@@ -28,13 +28,12 @@
 .Os
 .Sh NAME
 .Nm uiduid
-.Nd print your device udid to stdout
+.Nd print your device's UDID
```
Comment (CRKatri) [inline uiduid.1:31]:
```suggestion
.Nd print the device's UDID
```

File: uiduid.1 [hunk 9]
```diff
@@ -28,13 +28,12 @@
...
 .Sh SYNOPSIS
 .Nm
 .Sh DESCRIPTION
-The
 .Nm
-utility will print the udid then exit.
+will print the UDID of your device, then exit.
```
Comment (CRKatri) [inline uiduid.1:36]:
```suggestion
will print the UDID of the device, then exit.
```

**Review by CRKatri — COMMENTED** [review]
You probably shouldn't refer to the reader in documentation. This is minor but I'm mentioning it because most manpages don't do this.

File: cfversion.1 (hunk 4 above)
Comment (TheRealKeto) [reply cfversion.1]: Done.

**Pull request merged**